Exec/science/nova/inputs_nova_t7

```diff
@@ -69,36 +84,46 @@ amr.v                 = 1       # verbosity in Amr.cpp
 
 
 # REFINEMENT / REGRIDDING
-amr.max_level       = 2            # maximum level number allowed
+amr.max_level       = 1            # maximum level number allowed
 amr.ref_ratio       = 4 2          # refinement ratio
-amr.regrid_int      = 2 2          # how often to regrid
-amr.max_grid_size   = 256
-amr.blocking_factor = 64
-amr.n_error_buf     = 2 2         # number of buffer cells in error est
-amr.refine_grid_layout = 0
 
 
+#if AMREX_SPACEDIM == 3
+  amr.max_grid_size   = 64
+  amr.blocking_factor = 32
```

Comment:
I was using `max_grid_size = 96` and `blocking_factor=16` on Frontier.  We should play around with these more.